docker-jans-shibboleth/scripts/entrypoint.sh

```
@@ -0,0 +1,30 @@
+    touch /app/.initialized
+fi
+
+python3 /app/scripts/healthcheck.py &
+
+cd /opt/jans/jetty/shibboleth-idp
+
+exec java \
+    -server \
+    -XX:+DisableExplicitGC \
+    -XX:+UseContainerSupport \
+    -XX:MaxRAMPercentage=${CN_MAX_RAM_PERCENTAGE} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)